### PR TITLE
Fixed some layout glitches.

### PR DESCRIPTION
**Pull request by Elland**
Including: 
- #220 
- #219 
- #218

### CODE REVIEW

File: Token/Controllers/Settings/ProfileEditController.swift [hunk 1]
```diff
@@ -40,21 +40,14 @@ open class ProfileEditController: UIViewController {
...
     }
 
-    lazy var toolbar: UIToolbar = {
-        let view = UIToolbar(withAutoLayout: true)
-        view.barTintColor = Theme.viewBackgroundColor
-        view.tintColor = Theme.tintColor
-        view.delegate = self
-
-        let space = UIBarButtonItem(barButtonSystemItem: .flexibleSpace, target: nil, action: nil)
+    lazy var navbar: UINavigationBar = {
```
Comment (yuliaveres) [inline Token/Controllers/Settings/ProfileEditController.swift]: private ?

File: Token/Controllers/Settings/ProfileEditController.swift [hunk 2]
```diff
@@ -172,6 +172,12 @@ open class ProfileEditController: UIViewController {
             }
         }
     }
+
+    @objc private func didTapView(sender: UITapGestureRecognizer) {
+        if sender.state == .recognized {
+            self.becomeFirstResponder()
```
Comment (yuliaveres) [inline Token/Controllers/Settings/ProfileEditController.swift]: what is the purpose of the gesture ? not resigning ?

Comment (Elland) [inline Token/Controllers/Settings/ProfileEditController.swift]: To resign, I need to know exactly which table view cell's text field is the active one. This way, I don't have to care, keeping everything decoupled. 🚀

**Comment by Elland** [comment]
@yuliaveres ready for review again. 🚀 

Doesn't fix every issue, but improves a lot.

File: Token/Controllers/Settings/ProfileEditController.swift [hunk 3]
```diff
@@ -72,56 +68,59 @@ open class ProfileEditController: UIViewController {
...
         super.viewDidLoad()
 
-        self.view.backgroundColor = Theme.viewBackgroundColor
+        self.view.backgroundColor = Theme.navigationBarColor
         self.addSubviewsAndConstraints()
 
         guard let user = TokenUser.current else { return }
         self.avatarImageView.image = user.avatar
+
+        let tapGesture = UITapGestureRecognizer(target: self, action: #selector(ProfileEditController.didTapView))
```
Comment (yuliaveres) [inline Token/Controllers/Settings/ProfileEditController.swift]: just curious, what happens in case just -> #selector(didTapView)

Comment (Elland) [inline Token/Controllers/Settings/ProfileEditController.swift]: I think it calls our method even if a subclass overloads it. But it's mostly a reflex, used to doing that in ObjC.

**Pull request merged**